x/tokenregistry/client/cli/query.go

```diff
@@ -162,18 +162,15 @@ func GetCmdGenerateEntry() *cobra.Command {
 			if permissionIBCImport {
 				permissions = append(permissions, types.Permission_IBCIMPORT)
 			}
-			var path string
 			var denom string
 			// base_denom is required.
 			// generate denom if path is also provided.
 			// override the IBC generation with --denom if specified explicitly.
 			// otherwise fallback to base_denom
-			if ibcChannelId != "" {
-				// normalise path slashes before generating hash (do this in MsgRegister.ValidateBasic as well)
-				path = "transfer/" + ibcChannelId
+			if ibcPath != "" {
```

Comment:
Right now, the scripts expect to use --token_channel_id, so we need to keep support for that, if ibc_path isn't specified (it should then just be set as it previously was).